### DISCUSSION TITLE
Digitech-XC0324

### DISCUSSION THREAD

**Discussion by kidshare** [merbanan/rtl_433#2246] [General]
Not sure why this sensor is on the "disabled by default" list, I have been using the plugin for a couple years now and it works fine. It's just a cheap little sensor that only does temperature. It is sold as a TS-WS-47 on aliexpress.

**Comment by zuckschwerdt** [comment]
This decoder is part of a tutorial, and more verbose than others.
See infos here https://github.com/merbanan/rtl_433/blob/master/src/devices/digitech_xc0324.c

I don't remember or see a solid reason for disabling it though.

**Reply by dockSquadron** [reply]
Excellent cheap sensor for temp and humidity. FT005TH. Shows up as the Digitech mentioned. $5.50. Have them all over the house, refrigerators, freezers, etc. 

https://www.amazon.com/dp/B009V0O092

**Reply by dockSquadron** [reply]
Sorry for the typo (after you made the code updates). Name should be Ambient Weather FT005TH rather than Ambient Weather FT005RH.

**Reply by zuckschwerdt** [reply]
I was confused but didn't check ;)

**Reply by zuckschwerdt** [reply]
Fixed in 8871f73

**Comment by zuckschwerdt** [comment]
Thanks for the reports. Default enabled with 3ff11b4f now.
